ConcordiumWallet/Views/Stake/Baking/ComissionSettings/BakerCommissionSettingsViewModel.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  BakerCommissionSettingsViewModel.swift
+//  ConcordiumWallet
+//
+//  Created by Milan Sawicki on 06/11/2023.
+//  Copyright © 2023 concordium. All rights reserved.
+//
+
+import Combine
+import Foundation
+
+extension NumberFormatter {
+    static var commissionFormatter: NumberFormatter {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .decimal
+        formatter.multiplier = 100
+        formatter.maximumFractionDigits = 3
```

Comment:
```suggestion
        formatter.minimumFractionDigits = 3
        formatter.maximumFractionDigits = 3
```

Wouldn't that give a more stable appearance?

Reply:
Actually the two last digits is always 0. So I think we should show just one digit?